fjord/feedback/tests/test_views.py

```diff
@@ -102,6 +103,23 @@ def test_valid_sad(self):
         eq_(u'Firefox', feedback.product)
         eq_(u'14.0.1', feedback.version)
 
+    @with_waffle('thankyou', True)
+    def test_thank_you(self):
+        """Thanks_sad template is used when appropriate"""
+        url = reverse('feedback', args=(u'firefox',))
+        r = self.client.post(url, {
+            'happy': 0,
+            'description': u"I want to know why Firefox doesn't make me sandwiches!",
```

Comment:
The sad thank you page requires a few things for it to get shown:

1. locale is en-US
2. # words > x
3. sad feedback

This test probably uses en-US, but it's not explicitly stated as such so if the test harness changes, this might break. Further, if the code this is testing isn't right and the test harness changes, we might not ever find out because this is not explicitly testing the en-US locale.

That should change.

Further, we need to make sure that changing any of those things returns the "normal" Thank You page.